packages/suite/src/services/coinjoin/config.ts

```diff
@@ -10,6 +11,6 @@ export const COINJOIN_NETWORKS: Record<string, any> = {
         baseBlockHeight: 0,
         baseBlockHash: '0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206',
         // client settings
-        middlewareUrl: `${REGTEST_URL}Cryptography/`,
+        middlewareUrl: `${MIDDLEWARE_URL}Cryptography/`,
```

Comment:
@szymonlesisz I assume this should be the only place where the URL for coinjoin middleware should be updated.

Reply:
Thanks for the idea, it is now implemented as suggested.